### PR TITLE
Issue 11/get_posts

### PR DESCRIPTION
**Pull request by claytonrv**
# Changes Description

Select the type of changes you're doing on project:

- [ ] **Feature**
- [X] **Change**
- [ ] **Fix**
- [ ] **Configuration**

# Solution Descripton

Created the integration between the frontend and the backend of the project. Now the post list is being retrieved from the Diana API. As we're now getting the data from the backend, some of the interfaces of the components changed to be accordingly the backend payload.

# Tests
Correction on tests:
- CurrentDayPost.test.js
- InformationDialog.test.js

# Task

Diana-web: [Task\_11](https://github.com/Try-tech-Labs/diana-web/issues/11)

### CODE REVIEW

**Comment by claytonrv** [comment]
**Changes preview**
https://user-images.githubusercontent.com/7401421/108282547-937a5880-7160-11eb-8ed2-e27898648bf9.mp4

File: src/constants/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const baseUrls = {
+    development:{
+        development: 'http://localhost:8000',
+    },
+    test:{
```
Comment (lucasmauro) [inline src/constants/index.js:5]: Is line 5 `NODE_ENV` and line 6 `REACT_APP_ENVIRONMENT`? Meaning that we could include in the future another test environment, e.g. staging, as shown below?
```
test:{
        development: 'http://localhost:8000',
        staging: 'https://staging.trytechlab.com/diana',
},
```
Or even production -> disaster recovery?
```
production:{
        production: 'https://trytechlab.com/diana',
        disasterRecovery: 'https://dr.trytechlab.com/diana',
},
```
If that is the case, I very much like it.

**Review by mspdl — APPROVED** [review]

File: src/constants/index.js (hunk 1 above)
Comment (claytonrv) [reply src/constants/index.js:5]: Exactly that! Until I've done this task, I knew only that we need a dev and a prod configurations, but once I've ran the tests it used a test environment. So I think that it's configurable, and we could do things like, run on a staging, qa, sandbox, disasterRecovery and so on. 🙂

Comment (lucasmauro) [reply src/constants/index.js:5]: Nice!

**Review by lucasmauro — APPROVED** [review]
😎 👍

**Pull request merged**